utils/addTrailingSlash.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Add a trailling slash to a URL if required (and the appropriate ENV var is set).
+ * @param {string} url - URL to which trailing slash should be added.
+ * @returns {string} - URL with trailing slash.
+ */
+export default function addTrailingSlash(url) {
+  const env = Object.keys(process.env).length ? process.env : window.__ENV__; // eslint-disable-line no-underscore-dangle
+
+  // Add a trailing slash, if relevant env var is configured.
+  if (env.CONFIG_FORCE_TRAILING_SLASHES) {
+    return (
+      '/' !== url[url.length - 1] &&
+      ! url.includes('.')
```

Comment:
don't quite understand why the url needs a `.` in it, but i'm sure it's cool

Reply:
Ah yeah, I should add a comment and test case for this actually, was intended to prevent trailing slashes being added to filepaths.